SwiftWisdom/Core/Geometry/Geometry.swift

```diff
@@ -86,6 +86,11 @@ public func random(range: Range<Int>) -> Int {
 }
 
 public func avg(numbers: CGFloat...) -> CGFloat {
+    return avg(numbers)
+}
+
+public func avg(numbers: [CGFloat]) -> CGFloat {
```

Comment:
I'm just going with what @cprime said. I can see both sides of this: If you've got 0 values, there is no average. But also if you want an average of nothing, that would be 0, right?

I just think it should be consistent between this and the `ip_mean()` function.